kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/QueryDataServiceIntegrationTest.java

```diff
@@ -743,6 +743,13 @@ public void testGetProcessInstancesWithOrderByClause() throws Exception {
 
     @Test
     public void testErrorHandlingFailedToSignal() throws Exception {
+        List<ExecutionErrorInstance> errors = processAdminClient.getErrors(CONTAINER_ID, false, 0, 100);
+        if (!errors.isEmpty()) {
+            String[] ids = errors.stream().map(ExecutionErrorInstance::getErrorId).toArray(String[]::new);
+            processAdminClient.acknowledgeError(CONTAINER_ID, ids);
+            errors = processAdminClient.getErrors(CONTAINER_ID, false, 0, 10);
+            assertEquals(0, errors.size());
```

Comment:
@mswiderski the test has failed on this assert so the only explanation is that there are more than 100 errors to ack probably. I will try it with 1000.